app/models/event_participation.rb

```diff
@@ -0,0 +1,60 @@
+class EventParticipation < ActiveRecord::Base
+  include Diaspora::Federated::Base
+  include Diaspora::Guid
+  include Diaspora::Relayable
+
+  belongs_to :event
+  belongs_to :author, :class_name => "Person", :foreign_key => :author_id
+
+  xml_attr :diaspora_handle
+  xml_attr :intention
+
+  validate :not_already_participated
+  validates :intention, presence: true
+
+  enum intention: [ :no, :maybe, :yes ]
```

Comment:
Please assign explicit values to guard against later mistakes when adding elements or reordering.